### ISSUE TITLE
Istio chart/istio-installer - Grafana dashboard(7642) is broken

### ISSUE THREAD

**Issue by BadLiveware** [rancher/rancher#37618]
The rancher [istio-installer ](https://github.com/rancher/istio-installer) Dockerfile [tries to download](https://github.com/rancher/istio-installer/blob/1.11.4-rancher1/scripts/get_grafana_dashboards.sh) the relevant istio dashboards from the [grafana.com api](https://grafana.com/api/dashboards/7642/revisions) it seems however that grafana does not enumerate more than 43 revisions, which causes the script to create an invalid dashboard 
```cmd
bash-5.1$ cat 7642.json
{
  "code": "InvalidArgument",
  "message": "Field is required: revision"
}
```

which causes grafana errors
```log
t=2022-01-28T13:18:24+0000 lvl=eror msg="failed to load dashboard from " logger=provisioning.dashboard type=file name=sidecarProvider file=/tmp/dashboards/7642.json error="Dashboard title cannot be empty"
```

**Comment by doflamingo721** [comment]
**Steps to validate:**
1. **Rancher server installation type**: Docker
2. **Rancher server version:** 2.6-head (e53e6f8)
2. Create any downstream cluster to test.
3. Install istio `100.2.1+up1.12.6`
4. Check the pod logs for grafana pod under cattle-monitoring-system namespace. 

Result:
The logs should not contain any form of errors.

**Comment by rishabhmsra** [comment]
Validated on rancher v2.6-head(c4d59fa)

Steps followed:
- Provisioned a k8s `v1.23.6-rancher1-1` downstream cluster -> ec2 node driver (1-cp, 1-etcd, 2-worker)
- Installed Monitoring chart `100.1.2+up19.0.3`
- Installed Istio : `100.2.1+up1.12.6`
- Checked the logs of grafana pod in cattle-monitoring-system namespace. There were no error.

Result:
- Verified the logs and there were no errors, hence closing the issue.

**Comment by rishabhmsra** [comment]
Re-opening this as the fix will also go in istio v1.13.3 as well, so this needs to be tested again on latest version.
